pkg/webhook/webhook.go

```diff
@@ -289,6 +290,23 @@ func (w *Config) buildFleetValidatingWebhooks() []admv1.ValidatingWebhook {
 			},
 			TimeoutSeconds: fiveSecondWebhookTimeout,
 		},
+		{
+			Name:                    "fleet.membercluster.validating",
+			ClientConfig:            w.createClientConfig(membercluster.ValidationPath),
+			FailurePolicy:           &failFailurePolicy,
+			SideEffects:             &sideEffortsNone,
+			AdmissionReviewVersions: admissionReviewVersions,
+			Rules: []admv1.RuleWithOperations{
+				{
+					Operations: []admv1.OperationType{
+						admv1.Create,
+						admv1.Update,
+					},
+					Rule: createRule([]string{clusterv1beta1.GroupVersion.Group}, []string{clusterv1beta1.GroupVersion.Version}, []string{memberClusterResourceName}, &clusterScope),
+				},
+			},
+			TimeoutSeconds: fiveSecondWebhookTimeout,
```

Comment:
```suggestion
			TimeoutSeconds: longWebhookTimeout,
```
rename "oneSecondWebhookTimeout" to shortWebhookTime

the current name does not provide much info and we could change the value anytime in the future.